### DISCUSSION TITLE
how to use other url to connect to openai

### DISCUSSION THREAD

**Discussion by raccoon-zhang** [sashabaranov/go-openai#651] [Q&A]
why can not I use the other url to connect to openai , is that not supported? I can not change this。
```
const chatCompletionsSuffix = "/chat/completions"
```

**Comment by Guaderxx** [comment]
that's weird, what do you mean you need other url to connect to openai, proxy?

**Reply by raccoon-zhang** [reply]
yes, that's  it. I can't  get an api key for my region, so I have to try some other way and I think it is not just for me.

**Reply by raccoon-zhang** [reply]
sorry，I make a stupid mistake, I can change the BaseUrl of Client, then I can do it,  thanks for your help.

**Reply by Guaderxx** [reply]
没事，你带的姓很明显，我可能不算很明显，没事关了就行。

> 要改库的话 fork 一下随便改就行，无所谓的
